at-quannguyen/W4/main.rb

```diff
@@ -0,0 +1,59 @@
+require './bbcnews.rb'
+require './cnnnews.rb'
+require './user.rb'
+
+bbc_news_01 = BbcNews.new(01, "BBC first news", "Hello world from BBC", "Say hello")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/5cea93c6775631aa2d46f69d0bc35804ff430463/at-quannguyen/W4/main.rb#L5"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>